dataserver.js

```diff
@@ -0,0 +1,29 @@
+'use strict'; // eslint-disable-line strict
+require('babel-core/register');
+
+const arsenal = require('arsenal');
+const config = require('./lib/Config.js').default;
+const logger = require('./lib/utilities/logger').logger;
+
+if (config.backends.data === 'file') {
+    /**
+     * Configure the file paths for data if using the file
+     * backend. If no path provided, uses data at the root of
+     * the S3 project directory.
+     */
+    const dataPath = process.env.S3DATAPATH ?
```

Comment:
Or we could pull the config from the same server if they are on shared storage available to both S3 and dmds, where the S3 directory where S3 and dmds are run would be the same.